### ISSUE TITLE
youtube codec error

### ISSUE THREAD

**Issue by nibirrayy** [scivision/PyLivestream#67]
**Describe the bug**
I am trying to stream multiple files with a mp4 as a image on YouTube but I am getting codec errors on the stream as soon as the audio file change

`Command : FileGlobLivestream /root/music/Lofi-House youtube -glob *.mp3 -loop -image /root/Nychthemeron-stream/src/output.mp4 `

**System Parameters**

* Operating system : Linux Ubuntu 
* FFmpeg version: ffmpeg version 3.4.8

**Comment by scivision** [comment]
Yes It's something I don't have control over. Audio streaming like this is kind of a hack.
